src/riak_kv_keys_fsm.erl

```diff
@@ -94,8 +102,26 @@ init(From={_, _, ClientPid}, [Bucket, ItemFilter, Timeout]) ->
     %% Construct the key listing request
     Req = req(Bucket, ItemFilter),
     {Req, all, NVal, 1, riak_kv, riak_kv_vnode_master, Timeout,
+     #state{from=From}};
+
+init(From={_, _, ClientPid}, [Table, ItemFilter, Timeout, DDL]) ->
```

Comment:
Reworked the 'code path' from `riak_client:list_keys/2-5` to `riak_kv_keys_fsm:init/2`, as part of https://github.com/basho/riak_kv/commit/77cd66e3b81b0f23b7053b6f38c72df89405ca01.